README.adoc

```diff
@@ -96,6 +92,36 @@ oc create \
 
 After the role is granted, switch back to a non-privileged user.
 
+Jaeger Agent can be configured to be deployed as a `DaemonSet` using a `HostPort` to allow Jaeger clients in the same node to discover the agent. In OpenShift, a `HostPort` can only be set when a special security context is set. A security context constraint allowing `DaemonSet` components to bind to a `HostPort` can be specified as follows:
```

Comment:
Shouldn't this text all be before the previous statement, switching back to the non-privileged user?

Security concern - isn't this potentially enabling non-privileged users to deploy daemonsets on host ports? Isn't there a way to just grant the permission to the Jaeger operator, and the non-privileged user is then only able to deploy a Jaeger instance, which indirectly deploys daemonset with hostport?

Reply:
> Shouldn't this text all be before the previous statement, switching back to the non-privileged user?

Possibly. The reasoning is that users will *only* need to read on if they plan to allow agents as daemonsets.

> Security concern - isn't this potentially enabling non-privileged users to deploy daemonsets on host ports?

Yes, and good question. Would you know who might be able to help us here?